src/webots/gui/WbRobotWindow.cpp

```diff
@@ -169,7 +171,19 @@ void WbRobotWindow::show() {
 }
 
 QString WbRobotWindow::formatUrl(const QString &urlString) {
-  const QUrl &url(urlString);
+  static QString repoUrl;
+  if (repoUrl.isEmpty()) {
+    // forge the GitHub URL to the current version of Webots
+    const QString webotsVersion = WbApplicationInfo::version().toString().replace(" revision ", "-rev");
+    repoUrl = "https://raw.githubusercontent.com/cyberbotics/webots/" + webotsVersion + "/";
```

Comment:
Unfortunately, this will not work until we create the version tag: it won't work for the night builds for example.

Reply:
This link is automatically converted by Webots to the local file if the path found matches the current version.
So it works also before the tag is creates and even in the nightly builds.